docs/getting-started-guides/scratch.md

```diff
@@ -0,0 +1,727 @@
+  - The release contains files such as `./kubernetes/server/bin/kube-apiserver.tar` which
+    can be converted into docker images using a command like
+    `tar -C kube-apiserver -c . | docker import - kube-apiserver`
+  - *TODO*: test above command.
+
+For
+- Use images hosted on Google Container Registry (GCR), such as `gcr.io/google_containers/etcd:2.0.12`
+- Use images hosted on [Docker Hub](https://registry.hub.docker.com/u/coreos/etcd/) or [quay.io](https://registry.hub.docker.com/u/coreos/etcd/)
+- Use etcd binary included in your OS distro.
+- Build your own image
+  - You can do: `cd kubernetes/cluster/images/etcd; make`
+
+We recommend that you use the etcd version which is provided in the kubernetes binary distribution.   The kubernetes binaries in the release
+were tested extensively with this version of etcd and not with any other version.
+The recommended version number can also be found as the value of `ETCD_VERSION` in `kubernetes/cluster/images/etcd/Makefile`.
+
+The remainder of the document assumes that the image identifiers have been chosen and stored in corresponding env vars.  Examples (replace with latest tags and appropriate registry):
+  - `APISERVER_IMAGE=gcr.io/google_containers/kube-apiserver:$TAG`
+  - `SCHEDULER_IMAGE=gcr.io/google_containers/kube-scheduler:$TAG`
+  - `CNTRLMNGR_IMAGE=gcr.io/google_containers/kube-controller-manager:$TAG`
+  - `ETCD_IMAGE=gcr.io/google_containers/etcd:$ETCD_VERSION`
+
+### Security Models
+
+There are two main options for security:
+ 1. Access the apiserver using HTTP.
+   - Use a firewall for security.
+   - This is easier to setup.
+ 1. Access the apiserver using HTTPS  
+   - Use https with certs, and credentials for user.
+   - This is the recommended approach.
+   - Configuring certs can be tricky.
+
+If following the HTTPS approach, you will need to prepare certs and credentials.
+
+#### Preparing Certs
+You need to prepare several certs:
+  - The master needs a cert to act as an HTTPS server.
+  - The kubelets optionally need certs to identify themselves as clients of the master, and when
+    serving its own API over HTTPS.
+
+Unless you plan to have a real CA generate your certs, you will need to generate a root cert and use that to sign the master, kubelet, and kubectl certs.
+  - see function `create-certs` in `cluster/gce/util.sh`
+  - see also `cluster/saltbase/salt/generate-cert/make-ca-cert.sh` and
+    `cluster/saltbase/salt/generate-cert/make-cert.sh`
+
+You will end up with the following files (we will use these variables later on)
+    - `CA_CERT`
+      - put in on node where apiserver runs, in e.g. `/srv/kubernetes/ca.crt`.
+    - `MASTER_CERT`
+      - signed by CA_CERT
+      - put in on node where apiserver runs, in e.g. `/srv/kubernetes/server.crt`
+    - MASTER_KEY 
+      - put in on node where apiserver runs, in e.g. `/srv/kubernetes/server.key`
+    - KUBELET_CERT
+      - optional
+    - KUBELET_KEY
+      - optional
```

Comment:
This blurb is not formatted correctly in markdown. I think you might need to unindent once. Also the last three envvars don't have ``.